sqlglot/tokens.py

```diff
@@ -152,6 +152,7 @@ class TokenType(AutoName):
     UPDATE = auto()
     VALUES = auto()
     VIEW = auto()
+    WINDOW = auto()
```

Comment:
It seems like I don't fully understand the logic behind the tokenizer. I see that we added a new token type but I don't see a mapping for it in the Tokenizer's `KEYWORDS`.

Reply:
it's to make expression and token duck typed

Reply:
because expression needs to have a corresponding token_type

Reply:
maybe this should be changed?

Reply:
Not a blocker for this PR but perhaps worth revisiting at some point.